File: tests/method_test.py
#!/usr/bin/env python
import angle
from tests.parser_test_helper import *


class MethodTest(ParserBaseTest,unittest.TestCase):
    

    def test_result(self):
        parse('alias show = puts;show 3')
        assert_equals(result(), '3')
        parse('how to test:show 3;ok')
        assert(bigger_than(0, len(methods(), )))
        assert_equals(body(methods['test'], ), 'show 3;')
        parse('test')
        assert_equals(result(), '3')

File: angle.js/test/method.test.js
#!/usr/bin/env python;
// var angle = require('angle');



class MethodTest extends (ParserBaseTest) {
	test_result() {
		parse(`alias show = puts;show 3`);
		assert_equals(result(), '3');
		parse(`how to test:show 3;ok`);
		assert(bigger_than(0, len(methods(),)));
		assert_equals(body(methods['test'],), 'show 3;');
		parse(`test`);
		assert_equals(result(), '3');
	}
}